src/scenes/FunderReport/MetricBox.tsx

```diff
@@ -71,6 +71,7 @@ const MetricBox = ({ metric, index, year, quarter, length }: MetricBoxProps) =>
           </Typography>
           <Typography fontSize={'24px'} fontWeight={600}>
             {metric.target !== undefined && numberFormatter.format(metric.target)} {metric.target ? addPercentSign : ''}
+            {metric.target !== undefined ? metric.unit : ''}
```

Comment:
Should this be `metric.unit !== undefined` ? Same in line 83

Reply:
No in this case I'm only wanting to display the unit if the target is defined, because otherwise the text would just be the unit which could be confusing!